chatlas/__init__.py

```diff
@@ -22,6 +23,7 @@
 
 __all__ = (
     "ChatAnthropic",
+    "ChatAuto",
```

Comment:
I think we'll want to add this to `_quarto.yml` and the `README.md`?

Reply:
that too